### PR TITLE
[WIP] Add HTTPS scheme requirement for redirects

### PR DESCRIPTION
**Pull request by claudijd**
When visiting sso.mozilla.com via curl, I get an HTTP redirect in the chain.  I believe this fix could address this issue.

Additionally, I think it would be better if the hostname was more fluid and variablized, perhaps someone more familiar could point out what value this should be is so I can sub it in with a format call?

### CODE REVIEW

File: dashboard/app.py [hunk 1]
```diff
@@ -100,7 +100,8 @@ def favicon():
 
 @app.route("/")
 def home():
-    return redirect("/dashboard", code=302)
+    redirect_url = "https://sso.mozilla.com/dashboard"
```
Comment (andrewkrug) [inline dashboard/app.py]: This won't work because we host at multiple URLs.  Must inherit the base url from the CONFIG object in this string.

Comment (claudijd) [reply dashboard/app.py]: Ah, so steal from app.config.get("SERVER_NAME"), thanks!

Comment (andrewkrug) [reply dashboard/app.py]: This setting should also only be applied in the production env.

Comment (claudijd) [reply dashboard/app.py]: I'll mimic the logic here (but invert the conditional)

https://github.com/mozilla-iam/sso-dashboard/blob/master/dashboard/app.py#L263-L267

Comment (claudijd) [reply dashboard/app.py]: Fix should now only affect sso.mozilla.com

**Comment by gene1wood** [comment]
@claudijd or @andrewkrug do you know why this was closed. Checking and this issue (of the redirect to http) is still present

**Comment by andrewkrug** [comment]
It's natively settable on the flask server itself.  No need for a code based solution.

**Comment by floatingatoll** [comment]
Fixed by #437.

**Pull request closed without merging**